src/parser/parser_support.c

```diff
@@ -10968,39 +10968,199 @@ pt_get_server_name_list (PARSER_CONTEXT * parser, PT_NODE * node, void *arg, int
   return node;
 }
 
+#if defined (ENABLE_UNUSED_FUNCTION)
```

Comment:
For readability, how about setting ENABLE_UNUSED_FUNCTION separately for each function?